controlplane/pkg/prefix_pool/prefixpool.go

```diff
@@ -435,6 +436,43 @@ func fromIP(ip net.IP) (*big.Int, int) {
 	return val, 128
 }
 
+func min(a, b int) int {
```

Comment:
Should this be part of our `tool` or `utils`? Looks pretty generic. Or maybe there is a package that already provides it?

Reply:
That's funny but I found only `math.Min` that accepts `float` :) Moving this func to specific package is good idea